python/tvm/rpc/proxy.py

```diff
@@ -319,7 +319,7 @@ def _regenerate_server_keys(self, keys):
         new_keys = []
         # re-generate the server match key, so old information is invalidated.
         for key in keys:
-            rpc_key, _ = key.split(":")
+            rpc_key, _ = base.split_random_key(key)
             handle = self._server_pool[key]
             del self._server_pool[key]
             new_key = base.random_key(rpc_key + ":", keyset)
```

Comment:
Should we also update this line? Because variable `keyset` is not on the right place in the list of arguments and I expect that after first iteration we will have something like that: `127.0.0.1:5555::123456`.

Reply:
thanks. updated.